clever.go

```diff
@@ -150,6 +150,13 @@ type School struct {
 	SisID        string `json:"sis_id"`
 	StateID      string `json:"state_id"`
 	MdrNumber    string `json:"mdr_number"`
+	Principal    Principal
```

Comment:
I think it's best to be explicit, and, do we want to `omitempty`?  For example ````json:"principal,omitempty"````, because its not required?  (I guess we don't do that for `Location`, so probably can disregard.)